### DISCUSSION TITLE
Raspberry Pi - Release i20220904

### DISCUSSION THREAD

**Discussion by rsa2010** [rern/rAudio#1086] [Fixed Issues]
Let me start by saying I like this player very much.

However I found some problems with fresh install on rpi.
When I add a web radio station, the UI crashes (JSON invallid) I tracked it down to a missing "}" in the file /srv/httpd/data/mpd/counts.

After adding a station the counts file contained: `{ "webradio": 5`
I've manually editted the file (adding `}` at the end) and UI was correctly working again.

**Comment by ceejayemm** [comment]
Try updating to the latest release (currently 20221014) via Cog Menu / Addons as  there have been a number of fixes of this type since the image you downloaded.

Chris
